src/components/DropDown/story.js

```diff
@@ -28,103 +76,130 @@ class DropDownExample extends Component {
           Dropdown
         </h3>
         <p>
-          The dropdown itself does not have any background color so it is important to
-          position it how you like on whichever background you prefer. That is why
-          there are two shade props <code>light</code> and <code>dark</code>. The shade is only to do with the color of the text. So if you place the dropdown over a dark background choose the light shade for white text and vice versa.
+          DropDown component can behave differently depending on purpose it will be used. With <code>showOnHover</code> prop enabled,
+          it will show and hide menu when your mouse pass by.
+          <br />
+          <code>pullRight</code> and <code>direction</code> can also be used to control alignment and direction in which dropdown will open.
```

Comment:
The props <code>pullRight</code> and <code>direction</code> may be used to control alignment and direction in which dropdown will open.